src/test/user.test.js

```diff
@@ -67,16 +66,15 @@ describe('PUT REQUESTS', () => {
       email: faker.internet.email(),
       firstName: faker.name.firstName(),
       lastName: faker.name.lastName(),
-      country: faker.address.country(),
       gender: 'male',
       street: 'ajayi estate',
       city: faker.address.city(),
-      state: faker.address.state(),
+      state: 'Enugu',
       birthdate: faker.date.past(),
-      phoneNumber: faker.phone.phoneNumber()
+      phoneNumber: faker.phone.phoneNumber(),
     };
     const response = await chai.request(server).put(`/api/users/profile/${id}`).send(user)
-      .set('authorization', `Bearer ${token}`);
+      .set('Cookie', `token=${token};`);
```

Comment:
Adding the semicolon at the end of the token may throw an error
```suggestion
      .set('Cookie', `token=${token}`);
```

Reply:
@emmanuelnwankwo i'll merge this code, please change it on your rebases since it's such a minor instance.